WordPress/Classes/System/WordPressAppDelegate.swift

```diff
@@ -615,17 +615,29 @@ extension WordPressAppDelegate {
 extension WordPressAppDelegate {
 
     var currentlySelectedScreen: String {
+        guard let rootViewController = window?.rootViewController else {
+            return String()
```

Comment:
If I understand correctly, this would be a bad scenario and we don't expect this to be called.

Reply:
That's true. For now, I've added a `DDLogInfo` call in https://github.com/wordpress-mobile/WordPress-iOS/pull/19677/commits/805032b7f6de604f8cb51f27065599b03c701897 so we can know in some way if this has been called incorrectly.